interpreter/binary/decode.ml

```diff
@@ -200,6 +200,74 @@ let memop s =
   let offset = vu32 s in
   Int32.to_int align, offset
 
+let atomic_prefix s =
```

Comment:
Nit: perhaps `atomic_instr`, since the function names all refer to what's being decoded, not what preceded it.